scripts/bin/update-packages

```diff
@@ -194,9 +276,12 @@ _update_dependencies() {
 echo "INFO: Running update for: $*"
 
 if [[ "$1" == "@all" ]]; then
+	_fetch_and_cache_tags
 	for repo_dir in $(jq --raw-output 'del(.pkg_format) | keys | .[]' "${TERMUX_SCRIPTDIR}/repo.json"); do
 		for pkg_dir in "${repo_dir}"/*; do
+			_D=`date +%s%N | cut -b1-13`
```

Comment:
Consider renaming for clarity, maybe `_unix_`
Also [prefer `$()` over backtick command substitution](https://www.shellcheck.net/wiki/SC2006).
```suggestion
			_unix_milli="$(date +%s%N | cut -b1-13)"
```